recipe/meta.yaml

```
@@ -25,7 +25,7 @@ requirements:
   run:
     - ansi2html
     - h5py >=2.9
-    - ipykernel <=6.8 | >=16.5.2
+    - ipykernel <=6.8 | >=6.15.2
```

What about this instead?

```suggestion
    - ipykernel >=6.15.2
```

---

I am trying to avoid forcing upgrades when not strictly necessary. For example, I made backwardly-compatible changes to the Matplotlib code because v3.6.0 requires Python 3.8, which could break some distributions. I don't know if ipykernel v6.15.2 would cause any such dependency issues, but I try to minimize potential problems if possible.